### DISCUSSION TITLE
How to control thread count at android native?

### DISCUSSION THREAD

**Discussion by bykim555** [apache/incubator-mxnet#19728] [Q&A]
OS: Ubuntu 18.04 64bit LTS
Anaconda3-2019.03-Linux-x86_64
conda env used, Python: 3.6.8

git clone --recursive https://github.com/apache/incubator-mxnet.git
Tried to build version mxnet1.8.0.rc2
ci/build.py -p android_armv7. And build success.

I wish single thread. I tried code next...

#include <dmlc/omp.h>
omp_set_num_threads(1); 

But i can't controlled thread count. 
Could you help me please.

**Comment by leezu** [comment]
Have you tried setting the `OMP_NUM_THREADS` environment variable at runtime? You can also build without openmp support by specifying `-DUSE_OPENMP=0` to cmake.
